.github/workflows/commit-queue.yml

```diff
@@ -33,13 +33,24 @@ jobs:
       - name: Get Pull Requests
         id: get_mergeable_prs
         run: >
-          numbers=$(gh pr list \
+          prs=$(gh pr list \
                   --repo ${{ github.repository }} \
                   --base ${{ github.ref_name }} \
                   --label 'commit-queue' \
                   --json 'number' \
+                  --search "created:<=$(date --date="2 days ago"  +"%Y-%m-%dT%H:%M:%S%z")"
                   -t '{{ range . }}{{ .number }} {{ end }}' \
                   --limit 100)
+          fast_track_prs=$(gh pr list \
+                  --repo ${{ github.repository }} \
+                  --base ${{ github.ref_name }} \
+                  --label 'commit-queue' \
+                  --label 'fast-track' \
+                  --json 'number' \
+                  --search "created:<=$(date --date="2 days ago"  +"%Y-%m-%dT%H:%M:%S%z")"
+                  -t '{{ range . }}{{ .number }} {{ end }}' \
+                  --limit 100)
+          numbers=echo $prs'\n'$fast_track_prs | jq -s 'add'
```

Comment:
That shouldn't  be an issue, PRs landing after 2 days don't need the `fast-track` label anyway, the workaround is to remove the fast-track label if we're landing a PR after it's been open for two days.

Reply:
Ideally we don't need a workaround. Furthermore `| jq -s 'add'` doesn't really do what we expect here.


```sh
prs="1\n2"   
fast_track_prs="2\n3"
echo $prs'\n'$fast_track_prs | jq -s 'add'
8
```

Reply:
I think this would work?

```
echo $prs'\n'$fast_track_prs | jq -s 'unique | join("\n")'
```

```
prs="1\n2"
fast_track_prs="2\n3"
echo $prs'\n'$fast_track_prs | jq -s 'unique | join("\n")'
"1\n2\n3"
```